app/components/ingredient-list-edit.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  ingredients: null,
+  allIngredientOptions: null,
+  // what options are in the dropdown should not contain ingredients 
+  // already selected
+  currentIngredientOptions: Ember.computed('allIngredientOptions.[]', 'ingredients.[]', {
+    get() {
+      let allIngredients = this.get('allIngredientOptions');
+      let ingredients = this.get('ingredients');
```

Comment:
`let` is new in es6, so it's native javascript. It is indeed a new way of declaring variables, but mozilla will be able to better explain than I (and there's more links below): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

I was not able to use es6 in my last job, so I am still used to writing `var`, but we almost never, ever want `var`. You almost always want `let` to avoid scoping issues. We should probably make a card to change any `var` to `let`.

Here's some more discussion, I would recommend googling around and reading up more on it: http://programmers.stackexchange.com/questions/274342/is-there-any-reason-to-use-the-var-keyword-in-es6

In addition to `let`, there is also `const`: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

but honestly I am not keen on `const` as I think the name is misleading. Generally, I assume constants cannot be changed, but es6 `const`'s value can be changed, and using `const` where the first value it catches is somehow `undefined` can really give you a headache.

https://hacks.mozilla.org/2015/07/es6-in-depth-let-and-const/